content/language_bindings_and_conformance.tex

```diff
@@ -11,9 +11,10 @@
 implementing the interfaces using macros is strongly discouraged as this could
 severely limit the use of external profiling tools and high-level compiler
 optimizations. An \openshmem program should avoid defining routine names,
-variables, or identifiers with the prefix \shmemprefix (for \Clang{} and
-\Fortran), \shmemprefixC (for \Clang) or with \openshmem \ac{API} names.
+variable names, or identifiers with the prefix \shmemprefix{} (for \Clang{} and
+\Fortran), \shmemprefixC{} (for \Clang) or with \openshmem \ac{API} names.
 
 All \openshmem extension \ac{API}s that are not part of this specification must
 be defined in the \FUNC{shmemx.h} include file. These extensions shall use the
-\FUNC{shmemx\_} prefix for all routine, variable, and constant names.
+\FUNC{shmemx\_} prefix for routine names and variable names and the
```

Comment:
I think @jdinan's point is that, as proposed, the `SHMEMX_` prefix is reserved for constants (not variables or functions). Perhaps @BryantLam could use the term "integer constant expressions" or "preprocessor constants" to distinguish between `const int shmemx_*` constants and a constant suitable for evaluation by the preprocessor.